libcudacxx/test/stdpar/CMakeLists.txt

```
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.21)
+
+# NOTE: this is build outside of the libcu++ test harness
+project(CCCL_STDPAR_TESTS LANGUAGES CXX)
+
+# Enable testing for the project
+enable_testing()
+
```

I still think we should add this:

```suggestion

if (NOT CMAKE_CXX_COMPILER_ID STREQUAL NVHPC)
  message(FATAL_ERROR "The stdpar tests require nvc++ for CMAKE_CXX_COMPILER.")
endif()

```